app/models/maintenance_stats/queries/query_utils.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module MaintenanceStats
+  module Queries
+    class QueryError < StandardError; end
+
+    class QueryUtils
+      def self.check_for_graphql_errors(result)
+        if result.data.nil?
+          raise QueryError, "No data found in result"
```

Comment:
Are these worth logging so that we can track them over time? If so, might also be worth including more details to help with debugging.

Reply:
Updated the error checking a bit to put some structured logs in with the error messages and the repository name as fields.